learningequality/ka-lite
# Make update script call into setup (which detects it should do an update)

Fixes #962, #999, #1035

To test:
- Pull this code
- Add some videos through the video update page.
- Run `./manage.py setup` and do a fresh setup (say yes, when it asks you if you want to delete your database)
- Run `./manage.py katest unregister` to delete all registration info (log in through website as admin to verify it asks for registration information)
- Run `./manage.py update`
- Check that:
  - No errors
  - When you load the server as admin, it doesn't ask for registration info
  - All videos show as available in the webpages.

@wangguan59 could you try this out?  A little complicated, but we did this kind of testing together back in the "install_wizard" days :)
